Formula/zli.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class Zli < Formula
+  desc "BastionZero cli"
+  homepage "https://www.bastionzero.com"
+  url "https://github.com/bastionzero/zli/releases/download/4.9.0/zli-4.9.0.tar.gz"
+  sha256 "553b7a82a1b4b5015acc9bbe645afcca1fb351e92dbfe6514d7355e8df145136"
+  license "Apache-2.0"
+  head "https://github.com/bastionzero/zli.git"
+
+  bottle :unneeded
+  depends_on "node@12"
+
+  def install
+    system "npm", "install", *Language::Node.local_npm_install_args
+    system "npm", "run", "release"
+    rm "./bin/zli-win.exe"
+    rm "./bin/zli-linux"
+    mv "./bin/zli-macos", "./bin/zli"
+    bin.install "./bin/zli"
+  end
+
+  test do
+    system "zli", "help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula